.yarnrc.yml

```diff
@@ -0,0 +1,12 @@
+nodeLinker: node-modules
+
+plugins:
+  - path: .yarn/plugins/@yarnpkg/plugin-constraints.cjs
+    spec: "@yarnpkg/plugin-constraints"
+  - path: .yarn/plugins/@yarnpkg/plugin-interactive-tools.cjs
+    spec: "@yarnpkg/plugin-interactive-tools"
+  - path: .yarn/plugins/@yarnpkg/plugin-postinstall.cjs
+    spec: "https://raw.githubusercontent.com/gravitywelluk/yarn-plugin-postinstall/master/bundles/%40yarnpkg/plugin-postinstall.js"
+
+yarnPath: .yarn/releases/yarn-3.1.0.cjs
```

Comment:
Nit: 3.1.1 is latest I think

Reply:
I know. I thought we'd just let bots... I'll change it.